aiida_quantumespresso/calculations/__init__.py

```diff
@@ -332,7 +331,7 @@ def _generate_PWCPinputdata(cls, parameters, settings, pseudos, structure, kpoin
                 )
 
             kind_names.append(kind.name)
-            atomic_species_card_list.append('{} {} {}\n'.format(kind.name.ljust(6), kind.mass, filename))
+            atomic_species_card_list.append(f'{kind.name.ljust(6)} {kind.mass} {filename}\n')
```

Comment:
Should we change this to ``{kind.name:<6}``? Maybe it's just me, but I'm much more familiar with that way of specifying the format length.

Reply:
Me too, but I will leave that to another PR